Basic-Car-Maintenance/Shared/Dashboard/Views/DashboardView.swift

```diff
@@ -10,6 +10,10 @@ import SwiftUI
 struct DashboardView: View {
     
     @StateObject private var viewModel: DashboardViewModel
+    @State private var isShowingEditView = false
+        @State private var selectedMaintenanceEvent: MaintenanceEvent = MaintenanceEvent(
+            title: "", date: Date(), notes: ""
+        )
```

Comment:
`selectedMaintenanceEvent` this should actually be optional rather than creating an object, because we're not always selecting an event